src/components/Index.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import '../styles/sign.css'
+import SignIn from "./SignIn";
+import SignUp from "./SignUp";
+
+
+const Index = () => {
+    const [sign, setSign] = useState(false);
```

Comment:
I would recommend to name it ``` signUpView & setSignUpView ``` since it indicate weather the SignUp component is shown or not